docs/guides/shell-completion.md

```diff
@@ -2,39 +2,42 @@
 title: Enable Completion Script
 ---
 
-# Enable shell completion
+# Enabling shell completions
 
-Below is example steps to enable shell completion feature for `bearer` cli:
+Completions make it easier to use bearer on the command line. In this guide we will cover how to set everything up.
 
-## 1. Know your current shell
+## Discover your current shell
 
 ```bash
 $ echo $SHELL
-/bin/zsh # For this example it is zsh, but will be vary depend on your $SHELL, maybe /bin/bash or /bin/fish
+/bin/zsh
 ```
+In our example it's zsh, but will be vary depend on your $SHELL.
 
-## 2. Run `completion` command to get sub-commands
+## Test the completion scripts
 
-``` bash
-bearer completion -h
-```
+We currently support `zsh` `bash` and `fish`. To load completions in your current shell session you can run the following:
 
-Generate the autocompletion script for the zsh shell.
+```bash
+source <(bearer completion zsh); compdef _bearer bearer
+```
 
-If shell completion is not already enabled in your environment you will need
-to enable it.  You can execute the following once:
+Now you can test completions:
```

Comment:
```suggestion
Now you can test completions. Type `bearer` into your terminal and hit the tab button on your keyboard:
```

Reply:
will use this but remove the shell example